sdk/core/core-http/lib/util/xml.browser.ts

```diff
@@ -94,19 +96,19 @@ function domToObject(node: Node): any {
   return result;
 }
 
-// tslint:disable-next-line:no-null-keyword
-const doc = document.implementation.createDocument(null, null, null);
 const serializer = new XMLSerializer();
 
 export function stringifyXML(obj: any, opts?: { rootName?: string }) {
   const rootName = (opts && opts.rootName) || "root";
-  const dom = buildNode(obj, rootName)[0];
+  // tslint:disable-next-line:no-null-keyword
+  const doc = document.implementation.createDocument(null, null, null);
+  const dom = buildNode(doc, obj, rootName)[0];
   return (
     '<?xml version="1.0" encoding="UTF-8" standalone="yes"?>' + serializer.serializeToString(dom)
   );
 }
 
-function buildAttributes(attrs: { [key: string]: { toString(): string } }): Attr[] {
+function buildAttributes(doc: any, attrs: { [key: string]: { toString(): string } }): Attr[] {
```

Comment:
`doc` should be strongly typed in the `buildAttributes` and `buildNode` functions

Reply:
Updated